### ISSUE TITLE
Support for multiple contexts concurrently

### ISSUE THREAD

**Issue by taxilian** [stern/stern#246]
**What would you like to be added**:

It would be amazing to have the ability to watch multiple contexts concurrently, e.g. `stern --context cluster1,cluster2,cluster3 -n prod mypodname`

**Why is this needed**:

I have three different clusters for redundancy and it would be amazing to be able to tail an application across all three of them.  I'll totally understand if you decide to reject this request, but it would be super handy, so I thought I'd throw the idea in in case it's not too hard ;-)

**Comment by zerowidth** [comment]
Agreed, I'm working with a similar constraint with multiple contexts for redundancy and this would be a huge help.

**Comment by evgkrsk** [comment]
It would be handy to have context matching by regexp, like in https://github.com/jonnylangefeld/kubectl-mc
```bash
$ kubectl mc -r . -- version
```
